### ISSUE TITLE
Problem with some workers at initiating 

### ISSUE THREAD

**Issue by pendoturbina** [doktor83/SRBMiner-Multi#269]
![Captura de pantalla 2023-05-09 225809](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/d4a8915f-01b4-4675-a7e4-
1d858acad97d)

Any suggestions? The miner works for me on some rigs and not on others, I tried reverting to 2.2.4 and it works fine, I have the problem in 2.2.5 on some rigs

![Captura de pantalla 2023-05-09 225926](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/9edd30b4-0dd7-4a4c-b965-5a6c1f855812)

**Comment by doktor83** [comment]
Check this : https://github.com/doktor83/SRBMiner-Multi/issues/267

**Comment by pendoturbina** [comment]
> Check this : #267

It was that! now are all working! 

![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/4b9dccba-98fa-4781-a697-2236eb10558d)


![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/ee0d4fcc-00ef-4b3a-bc6d-aafad7499be0)

I couldn't find where to read about it, I would like to understand more how this parameter affects the hashrate, because for example now I use the value "256" but lower the hashrate in almost all gpus from 1400hs to 1200hs. Do you have any recommendation? they are all RX580

**Comment by doktor83** [comment]
yeah,try 320

**Comment by pendoturbina** [comment]
With the new update cant get work the same rigs that have to push 256 or 320 intensity...

![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/5814c85a-a76c-4d56-9fc5-8815362e9df1)
![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/345c87c0-6ddd-45a1-a6c1-0e391fc96c01)
![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/3b8f8878-20f6-486a-bea0-687ee10819d4)

i dont now what to do at this point

**Comment by doktor83** [comment]
No idea because rx580 8g runs --gpu-intensity 368 without any issues for me. Can you run from console **--list-devices** and make a screenshot on one of the machines that have the issue ?

**Comment by pendoturbina** [comment]
![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/a6b572ae-fd70-4f0b-817f-f78e2c21a41d)

this is with the command list devices on one of the problematic machines

**Comment by doktor83** [comment]
Looks to me the first gpu is probably your problematic one. Try this **--gpu-id 1,2,3,4,5,6,7** , it should be ok even without using --gpu-intensity manually

**Comment by pendoturbina** [comment]
Now its working, i dont know wy is only the 0gpu thats unable to start the others? anywork around?

![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/fa0070fc-750f-4b61-8b50-55f7bd4871fe)

**Comment by pendoturbina** [comment]
or any way to run on all gpus but the fails ones leaves iddle?

**Comment by doktor83** [comment]
For some reason (you can see on the screenshot) that gpu reports it can allocate max 4GB, the others can allocate all 8GB.
Its your mining OS doing something wrong, maybe a fresh install would help, i don't know.
A workaround could be to skip those gpu's with --gpu-id, or to set a really low intensity on those cards .

**Comment by pendoturbina** [comment]
There is nothing to do with the bios o anything like above 4g deconding or something?, today upgrade hiveos to lastes and go with same problem the  "0" gpu not inicialincing...

**Comment by pendoturbina** [comment]
Good news, i tried a older version 2.2.4 and works on all gpus... but lower hashrate. something is broken or changed on init secuencies for 2.2.5 and 2.2.6

![image](https://github.com/doktor83/SRBMiner-Multi/assets/133072563/c30c32a5-cc96-4110-8bfb-611016c22792)

How can help you to workaround?

**Comment by pendoturbina** [comment]
Any toughts?

**Comment by doktor83** [comment]
Yes, i still dont think its a miner issue because even without running the miner (--list-devices), you can see that the AMD video driver reports a lower value for available buffer size for that gpu.

**Comment by pendoturbina** [comment]
Ir worked but one gpus less for that version 2.2.6 on 2.2.4 work for all gpus flawless...
